src/app.js

```diff
@@ -56,12 +66,14 @@ app.get('api/movies/:movieID/rating', async (req, res) => {
   // Placeholder, delete me...
 });
 
-app.get('api/movies/:movieId/reviews', async (req, res) => {
-  // Placeholder, delete me...
-});
+app.get('api/movies/:movieID/reviews', async (req, res) => {});
```

Comment:
Haha I'm confused, it looks as though this line of code is removed and added several times in the same PR, but anyways as I interpret it this line is not necessary since you have the same app.get just below...?

Reply:
Actually i dont know what happened, but i think something went a little crazy when i resolved the comment jesper made..